### ISSUE TITLE
Event userLoaded does not fire

### ISSUE THREAD

**Issue by liborpansky** [perarnborg/vuex-oidc#31]
Hi, first of all thank you for a job well done on the library!

I have an issue with latest version - after updating to 1.15+ event userLoaded fires just the first time after login, but the event does not trigger anymore after page refresh.

I'm using this event to set bearer header in Axios:
```
oidcSettings,
{ dispatchEventsOnWindow: true },
{
  userLoaded: user => ApiService.setHeader(user.access_token),
  userUnloaded: () => ApiService.setHeader(null),
  userSignedOut: () => ApiService.setHeader(null)
}
```

and then refresh token won't get set and all the API calls result in 401.

Is it correct behaviout? Maybe the modifications you made to check userManager before reauthenticating are correct and the event should not fire at all on page refresh - in this case I should change logic of token read for Axios, even though I don't have access to store in ApiService...

**Comment by perarnborg** [comment]
Thank you for taking the time to report an issue.

At first glance I would say that this is not correct behaviour, the event should fire when the user is loaded from the storage as well as after authentication. I have to dig into this before I can say for certain though. I would have thought that the oidc-client library would trigger the event in this case, but perhaps vuex-oidc is missing that. Almost certainly related to the changes made in v1.15 anyway.

**Comment by perarnborg** [comment]
Fixed in v1.15.2

Let me know if you still are having problems @liborpansky

**Comment by liborpansky** [comment]
Works as expected, thank you for the quick fix!
